src/core/compact_object.cc

```diff
@@ -707,6 +713,16 @@ void CompactObj::SetJson(JsonType&& j) {
   }
 }
 
+void CompactObj::SetSBF(uint64_t initial_capacity, double fp_prob, double grow_factor) {
+  if (taglen_ == SBF_TAG) {  // already json
+    *u_.sbf = SBF(initial_capacity, fp_prob, grow_factor, tl.local_mr);
+  } else {
+    SetMeta(SBF_TAG);
+    void* ptr = tl.local_mr->allocate(sizeof(SBF));
```

Comment:
Actually didn't know it automatically uses alignment 🤓 https://en.cppreference.com/w/cpp/memory/memory_resource/allocate